backend/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/RhosakEmailAggregator.java

```diff
@@ -15,7 +15,7 @@
     private static final String SERVICE_DISRUPTION_EVENT_TYPE = "disruption";
     private static final String CONTEXT = "context";
     private static final String EVENTS = "events";
-    private static final String PAYLOAD = "events";
+    private static final String PAYLOAD = "payload";
```

Comment:
good catch @gwenneg 

I'll be happy if you can give this file another round of review. 

Normally, I should have put some unit tests into it but I am still trying to find my way into the aggregator part.

Reply:
I had a look at the file. I'm not (yet 😄) an aggregator expert but it looks good to me.

Reply:
Thank you a lot @gwenneg